fjord/base/jinja2/includes/common.html

```diff
@@ -7,7 +7,7 @@
         <label for="language">{{ _('Other languages:') }}</label>
         <select id="language" name="lang" dir="ltr">
           {% for code, name in LANGUAGES|dictsort -%}
-            <option value="{{ code }}" {{ code|ifeq(LANG|lower, "selected") }}>
+            <option value="{{ code }}" {{ "selected" if code == LANG.lower() }}>
```

Comment:
It seems unnatural and unreadable to me Maybe there was a use case it was good at?